src/wq/tests/benchmark/driver.py

```diff
@@ -48,9 +49,9 @@ def _prep_monitoring_dir(self):
 
     def _start_docker_and_wait_for_services(self):
         self._docker.start()
-        sleep(10)
-        # TODO call QL's version endpoint rather than sleeping.
-        # If it's up, then Redis & DB backend are up to b/c of docker deps.
+        version_url = f"{QL_BASE_URL}/version"
+        r = requests.get(version_url)
+        r.raise_for_status()
```

Comment:
That's a good start, but there's still room for improvement :-)

The problem with this approach is that if QL starts after the `requests.get` call, the benchmark will exit. Ideally we should wait until we know QL is up or bail out after a max amount of secs. There's a function in `reporter.tests.utils` we can use to implement this flow. I'm sketching out how below in pseudo code.

```python
from reporter.tests.utils import wait_until

def _can_get_ql_version() -> bool:
    status = requests.get(version_url)
    return status == okay

def wait_for_ql():
    wait_until(_can_get_ql_version)

class TestScript:
    ...
    def _start_docker_and_wait_for_services(self):
        self._docker.start()
        wait_for_ql()
        # If QL is up, then Redis & DB backend are up to b/c of docker deps.
```